packages/java/endpoint/src/main/java/com/vaadin/hilla/EndpointInvoker.java

```diff
@@ -83,9 +83,8 @@ public class EndpointInvoker {
      * @param applicationContext
      *            The Spring application context
      * @param endpointObjectMapper
-     *            optional factory bean to override the default
-     *            {@link JacksonObjectMapperFactory} that is used for
-     *            serializing and deserializing request and response bodies Use
+     *            The object mapper to be used for serialization and
+     *            deserialization of request and response bodies. Use the
      *            {@link EndpointController#ENDPOINT_MAPPER_FACTORY_BEAN_QUALIFIER}
      *            qualifier to override the mapper.
```

Comment:
I would clarify a little bit the override hint, since the qualifier is not supposed to be put on mapper bean, but on a factory. Maybe something like


> To override the mapper, use the {@link EndpointController#ENDPOINT_MAPPER_FACTORY_BEAN_QUALIFIER} qualifier on a JacksonObjectMapperFactory bean definition.